teos/src/dbm.rs

```diff
@@ -970,6 +1177,44 @@ mod tests {
         assert_eq!(dbm.load_appointments(Some(locator)), appointments);
     }
 
+    #[test]
+    fn test_get_appointment_length() {
+        let dbm = DBM::in_memory().unwrap();
+
+        let user_id = get_random_user_id();
+        let user = UserInfo::new(AVAILABLE_SLOTS, SUBSCRIPTION_START, SUBSCRIPTION_EXPIRY);
+        let (uuid, appointment) = generate_dummy_appointment_with_user(user_id, None);
+
+        dbm.store_user(user_id, &user).unwrap();
+        dbm.store_appointment(uuid, &appointment).unwrap();
+
+        assert_eq!(
+            dbm.get_appointment_length(uuid).unwrap(),
+            appointment.inner.encrypted_blob.len()
+        );
```

Comment:
Ideally, a "random" transaction could be created by just modifying some of the transaction bits, such as value, prev_txid, ...

It may be worth adding something on these lines to the testing suite at some point.

Reply:
`prev_txid` is already random (used for encryption) but I don't think this affects the length of the encrypted bytes.

```
pub struct Transaction {
    /// The protocol version, is currently expected to be 1 or 2 (BIP 68).
    pub version: i32,
    /// Block number before which this transaction is valid, or 0 for valid immediately.
    pub lock_time: u32,
    /// List of transaction inputs.
    pub input: Vec<TxIn>,
    /// List of transaction outputs.
    pub output: Vec<TxOut>,
}
```
We can add more inputs/outputs to the transaction. A variable size `OP_RETURN` should do the trick.